File: fibonacci.py
def fibonacci(n):
    """Generate the first n Fibonacci numbers."""
    if n <= 0:
        return []
    if n == 1:
        return [0]

    fibs = [0, 1]
    for _ in range(2, n):
        fibs.append(fibs[-1] + fibs[-2])
    return fibs


if __name__ == "__main__":
    count = int(input("How many Fibonacci numbers? "))
    print(fibonacci(count))

File: fibonacci.js
function fibonacci(n) {
  if (n <= 0) return [];
  if (n === 1) return [0];

  const fibs = [0, 1];
  for (let i = 2; i < n; i++) {
    fibs.push(fibs[i - 1] + fibs[i - 2]);
  }
  return fibs;
}

// Run from command line: node fibonacci.js <count>
const count = parseInt(process.argv[2], 10) || 10;
console.log(fibonacci(count));
